docs/rst/manual/forward_model/index.rst

```diff
@@ -11,45 +11,229 @@ modelling tasks like e.g. relperm interpolation or water saturation, or to
 
 
-
-The forward model in the ert configuration file
------------------------------------------------
+The `FORWARD_MODEL` in the ert configuration file
+-------------------------------------------------
 
 The traditional way to configure the forward model in ERT is through the keyword
-`FORWARD_MODEL`. Each `FORWARD_MODEL` keyword will instruct ERT to run one
-particular executable, and by listing several `FORWARD_MODEL` keywords you can
+:code:`FORWARD_MODEL`. Each :code:`FORWARD_MODEL` keyword will instruct ERT to run one
+particular executable, and by listing several :code:`FORWARD_MODEL` keywords you can
 configure a list of jobs to run.
 
 
 The `SIMULATION_JOB` alternative
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-In addition to `FORWARD_MODEL` there is an alternative keyword `SIMULATION_JOB`
+In addition to :code:`FORWARD_MODEL` there is an alternative keyword :code:`SIMULATION_JOB`
 which can be used to configure the forward model. The difference between
-`FORWARD_MODEL` and `SIMULATION_JOB` is in how commandline arguments are passed
+:code:`FORWARD_MODEL` and :code:`SIMULATION_JOB` is in how commandline arguments are passed
 to the final executable.
 
 
-
 The runpath directory
 ---------------------
 
 Default jobs
 ~~~~~~~~~~~~
 
 It is quite simple to *install* your own executables to be used as forward model
-jobs in ert, but some common jobs are distributed with the ert/libres
+job in ert, but some common jobs are distributed with the ert/libres
 distribution.
 
-Reservoir simulation: eclipse
-.............................
+Reservoir simulation: eclipse/flow
+..................................
+
+.. code-block:: bash
+
+    EXECUTABLE script/ecl100
+    DEFAULT <VERSION> version
+    DEFAULT <NUM_CPU> 1
+    ARGLIST <ECLBASE> "--version=<VERSION>" "--num-cpu=<NUM_CPU>" "--ignore-errors"
+
+There are forward models for each of the simulators eclipse100, eclipse300 and
+flow, and they are all configured the same way. The version, number of cpu and
+whether or not to ignore errors can be configured in the configuration file
+when adding the job, as such:
+
+.. code-block:: bash
+
+    FORWARD_MODEL ECLIPSE100 <ECLBASE> "--version=xxxx"
 
 Reservoir modelling: RMS
 ........................
 
 File system utilities
 .....................
 
+There are many small jobs for copying and moving files and
+directories. The jobs are just thin wrappers around the corresponding
+unix commands, i.e. :code:`/bin/cp` and :code:`/bin/ln`, but we have
+tried quite hard to make them *safe* with error checking and "careful"
+default behaviour. The jobs are typically quite strict, i.e. if you
+try to remove a directory with the :code:`DELETE_FILE` job, or visa
+verse, nothing will happen. The actions performed will typically be
+logged to the :code:`xxx.stdout` files:
+
+.. code-block:: bash
+
+    Copying file 'file.txt' to 'target_backup.txt'
+
+And error messages will go to the :code:`xxx.stderr` files. None of
```

Comment:
This is not true, a non-zero return value will be picked up. This must be quite outdated!!!

Reply:
Probably, we’ll have to go through this a few times I guess. Thanks for notifying about this one though, I’ll get it updated! :)

Reply:
Given that you're providing a non-existing file, the forward model will be considered successful, and the following will be displayed in the `stderror` file of the job:

`File: '/path/to/file/delete_me' not exist - delete ignored`

Trying to delete a directory with the `DELETE_FILE` job, the forward model will fail with the typical `[000:0000-0000] FAILED COMPLETELY.` and traceback from `stderror` then displays:

```
Traceback (most recent call last):
  File "/work/install/ert/share/ert/forward-models/shell/script/delete_file", line 8, in <module>
    delete_file( file )
  File "/work/install/ert/lib/python2.7/site-packages/res/fm/shell/__init__.py", line 13, in delete_file
    Shell.deleteFile(filename)
  File "/work/install/ert/lib/python2.7/site-packages/res/fm/shell/shell.py", line 134, in deleteFile
    raise IOError("Entry:'%s' is not a regular file" % filename)
IOError: Entry:'delete_me' is not a regular file
```
I suppose that failing a job -> fails the forward model, is sensible. Is it just as sensible ignoring the delete if the file does not exist? What if the user has misspelled the name?

Reply:
I agree that this should be discussed, but we should then agree on consistent behaviour for all of the shell commands. Also, changes like this should be done with care...

Let's be happy for now if the documentation adheres to the current behaviour ;)